react-native/react/get-element-size-or-position-with-onLayout.mo.md

```diff
@@ -0,0 +1,81 @@
+# [MO] Get element size or position dynamically with onLayout
+
+## Owner: [Julien Nassar](https://github.com/juliennassar)
+
+## Control Points
+
+- Hide your dynamically sized (or positioned) component width a `style={{ opacity: 0 }}` during the first rendering of your component to avoid glitches.
+
+## Motivation
+
+If you need :
+- to create a custom loading bar display
+- a list of views without previous knowlegde of their size but want them to be all the same
+- just need to know the size or position of an element on your screen dynamically (whithout explicitly describing it the object's style)
+
+You must have access those variables during the runtime of your application.
+
+To do this, React offers a onLayout props in which you can pass a callback that will be executed after the rendering of a component.
+
+## Prerequisites
+
+A react/react-native app with components. We will take here the example of a loading bar like this one :
```

Comment:
A React Native app (the use of components is implicit).

This does not work on React as `onLayout` only exists in React Native components.